test/ssl-tests/14-groups.conf

```diff
@@ -2,54 +2,54 @@
 
 num_tests = 29
 
-test-0 = 0-curve-sect163k1
```

Comment:
I am not sure if renaming the tests and the test file is a good idea.  These are testing curves.  (Which are now, I guess, a subset of what a group is.)   I would prefer it as-is.